appengine/search/google-checks.xml

```diff
@@ -0,0 +1,223 @@
+<?xml version="1.0"?>
```

Comment:
This file is here just because I wanted to add two modules, `SuppressionCommentFilter` to the `Checker` section and `FileContentsHolder` to the `TreeWalker` section.

If you can add them to the configuration file in java-repo-tools, I can remove it, but if we remove it now, checkstyle complains.

Can you add them to the main config files, then I will rebase to it then everything will be fine?